config/formats.js

```diff
@@ -550,7 +551,7 @@ exports.Formats = [
 		name: "[Gen 7] Sketchmons",
 		desc: [
 			"Pok&eacute;mon gain access to one Sketched move.",
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587743/\">Sketchmons</a>",
+			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587743/\">Sketchmons Resources</a>",
```

Comment:
I think you meant to add the resources thread. Not rename the regular thread to resources.